ansible_collections/arista/avd/roles/eos_designs/doc/management-settings.md

```diff
@@ -17,6 +17,7 @@ local_users:
     role: < Specify a role for the user >
     no_password: < true | do not configure a password for given username. sha512_password MUST not be defined for this user. >
     sha512_password: "< SHA512 ENCRYPTED password >"
+    ssh_key: < SSHKEY string | Optional >
```

Comment:
Added by mistake (supported in eos_cli_config_gen not in eos_designs). Need to remove this knob.